src/main/java/org/jabref/gui/preferences/linkedfiles/LinkedFilesTabViewModel.java

```diff
@@ -77,7 +77,7 @@ public void setValues() {
         mainFileDirectoryProperty.setValue(filePreferences.getMainFileDirectory().orElse(Path.of("")).toString());
         useMainFileDirectoryProperty.setValue(!filePreferences.shouldStoreFilesRelativeToBibFile());
         useBibLocationAsPrimaryProperty.setValue(filePreferences.shouldStoreFilesRelativeToBibFile());
-        fulltextIndex.setValue(filePreferences.shouldFulltextIndexLinkedFiles());
+        fulltextIndex.setValue(!filePreferences.shouldFulltextIndexLinkedFiles());
```

Comment:
This change is wrong. It does not change the default value, it just changes the logic of how to represent the preferences in the dialog and now shows simply the opposite of the actual preferences value.
Plase note that changes to default values have to be made in JabRefPreferences.
But I believe that full text indexing is already disabled by default.